### PR TITLE
images now render in corresponding containers

### PR DESCRIPTION
**Pull request by Timilof**
Page is correct but requires a new button for displaying the preview popup

### CODE REVIEW

File: static/js/partials/render.js [hunk 1]
```diff
@@ -14,10 +14,11 @@ function newImages (images) {
 
       button.dataset.image = image[0]
       button.dataset.shot = 0
-
+// set comments here @jeroen
       const src = `../images/thumbnails/thumbnails_large/${image[0]}/${image[0]}_0.png`
       const img = element.image(src, '')
-
+// Tim placed a button inside the thing here so we can make ap preview button for each element
```
Comment (jeroentvb) [inline static/js/partials/render.js]: Is this done?

Comment (Timilof) [reply static/js/partials/render.js]: No, he is nog niet done yet

Comment (jeroentvb) [reply static/js/partials/render.js]:
```js
parentEl.appendChild(previewLink)
```

But I don't think having a `button` inside a `button` element is semantically correct.
Let's discuss this with @MartijnReeuwijk tomorrow.

Comment (MartijnReeuwijk) [reply static/js/partials/render.js]: tru maybe make it an A

Comment (MartijnReeuwijk) [reply static/js/partials/render.js]: Also styling with cach reminder

**Pull request merged**